cnames_active.js

```diff
@@ -3366,6 +3366,7 @@ var cnames_active = {
   "typed-ui": "pi-cubed.github.io/typed-ui",
   "typedi": "freshgum-bubbles.github.io/typedi",
   "typeless": "typeless.netlify.app",
+  "typerinth": "kartoffelchipss.github.io",
```

Comment:
```suggestion
  "typerinth": "kartoffelchipss.github.io/Typerinth",
```
If the repo is https://github.com/KartoffelChipss/Typerinth, we'll want the target here to include the repo name so we know where you contact you if needed